### DISCUSSION TITLE
Music control from taskbar preview pane

### DISCUSSION THREAD

**Discussion by LolloDev5123** [ppy/osu#21212] [Ideas] [answered]
It'd be cool to be able to control the idle osu! music playlist from the Windows taskbar, like Spotify

![immagine](https://user-images.githubusercontent.com/26120324/201476263-c194f04f-592b-427f-8b27-176241bfa767.png)

It'd essentially act like a shortcut to these controls:

![immagine](https://user-images.githubusercontent.com/26120324/201476281-49696899-eb31-48c1-bba1-33fb5ca55c85.png)

In technical .NET terms, we would need to register a `SystemMediaTransportControl`

**Comment by peppy** [comment, marked as answer]
Please use spotify to play music. osu! isn't intended to be used as a music player.
